source/developer/mobile-developer-setup.rst

```diff
@@ -14,6 +14,82 @@ If you run into any issues getting your environment set up, check the Troublesho
 Development Environment Setup
 ---------------------------------
 
+We strongly recommend the use of a computer that supports macOS to develop and build the Mattermost mobile apps as macOS is required to build the iOS app.
+
+Software Requirements
+---------------------------------
+
+Common
+~~~~~~~~~~~~~~~~~~~
+
+Irrespective of which platform you are going to develop and build, you'll need to install some prerequisite software. For macOS, we recommend using `Homebrew <https://brew.sh/>`_ as a package manager.
```

Comment:
Shorten `Irrespective of which platform you are going to develop and build, you'll need to install some prerequisite software.` to something like `Install the following software to develop and build the iOS or Android apps`. (maybe something a bit better, my wording is a little awkward)